nntrainer/tensor/tensor.cpp

```diff
@@ -266,6 +266,16 @@ Tensor Tensor::pow(float exponent) const {
   return apply([=](float in) { return powf(in, exponent); });
 }
 
+Tensor Tensor::getBatchSlice(unsigned int offset, unsigned int size) const {
```

Comment:
Size seems not used. Could you check this out?

Reply:
good finding, updated.